js/domUpdates.js

```diff
@@ -4,10 +4,12 @@ const domUpdates = {
   let namesAndScore = document.createElement('h1');
   namesAndScore.innerHTML = `<div class="name-styling">
   <h1 class="name">${player}</h1>
-  <h2 class="score">0</h2>
+  <h2 class="score ${player}">0</h2>
   </div>`;
   playerDisplay.append(namesAndScore);
   });
   }
 }
 
+// can we add different classes using interpolation to update the score of each player after their turn without changing every player?//
+
```

Comment:
Yes - I would make a class that's like `player-1`, `player-2`, `player-3`. Remember that `forEach` also gives you access to the index of each element, not just the element you're currently iterating on.

One other thing I notice about this file is that you're iterating over `game.players`. Your domUpdates file will not have access to the instance of your game unless you pass it in to the methods being called. e.g. your `setPlayerNames` method should take in the instance of your game as an argument. Otherwise `game.players` is going to be undefined.